### ISSUE TITLE
Dropdown list issue

### ISSUE THREAD

**Issue by nourhan-ashraf** [unstructuredstudio/zubhub#612]
**Describe the bug**
projects, languages and avatar dropdown lists appear above the button.

**Expected behavior**
It'll be better if the dropdown list appear below the button 

**Screenshots**
Projects dropdown list
![projects](https://user-images.githubusercontent.com/66208099/223888521-0fe93262-6c32-45a3-8c53-35852626c6b1.png)

Languages dropdown list
![lang](https://user-images.githubusercontent.com/66208099/223888536-6936bd9a-9b07-4b50-bc8d-ff533f20905f.png)

Avatar dropdown  list
![avatar](https://user-images.githubusercontent.com/66208099/223888544-97b5baea-af6b-4d01-85d3-c5dc2dc6cfba.png)

**Comment by KramStyles** [comment]
@nourhan-ashraf can I work on this if its approved?

**Comment by tuxology** [comment]
I tried to verify this issue. The issue is not with the dropdown list itself. This seems acceptable behavior to me. In the first screenshot the menu is placed correctly but moves ONLY after you resize the screen after dropdown list is loaded. We can keep this open but mark this low priority

**Comment by srish** [comment]
No longer valid issue with new UI changes
